CHANGELOG.rst

```diff
@@ -37,6 +37,8 @@ UNRELEASED
   * FIXED:     Reset `Software PLL` phase/frequency detector when digital clock
     becomes invalid to prevent incorrect error input to sigma-delta modulator
   * FIXED:     Incorrect values of bLockDelayUnits and bLockDelay in async mode
+  * FIXED:     Added mixer optimisation to speed up sample exchange between mixer and
```

Comment:
I would put this as a change since it the issue didnt arguably exist in a previous release.